VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -231,6 +233,7 @@ public void onClick(View v) {
             @Override
             public void onClick(View v) {
                 environment.getRouter().forceLogout(getActivity(), environment.getSegment(), environment.getNotificationDelegate());
+                SecurityUtil.clearUserData(MainApplication.application);
```

Comment:
Umm, invalidating tokens didn't have the special requirement for it to work only when a user interaction happens, so, I guess its correct as is.